app/guid-file/route.ts

```diff
@@ -54,7 +58,28 @@ export default class GuidFile extends Route {
                 this.intl.t('general.presented_by_osf'),
             language: this.metadata.language ? this.metadata.language : undefined,
         };
-        this.set('headTags', this.metaTags.getHeadTags(metaTagsData));
+
+        // Google Structured Data
+        const parentId = await model.target.get('id');
+        const jsonLD: object = await this.scriptTags.returnStructuredData(parentId);
+        const jsonString: string = Object.entries(jsonLD) ?
+            JSON.stringify(jsonLD) : JSON.stringify({ isAccessibleForFree : true });
+        const scriptTagData = {
+            type: 'application/ld+json',
+            content: jsonString,
+        };
+
+        const metaTags: HeadTagDef[] = this.metaTags.getHeadTags(metaTagsData);
+        const scriptTag: HeadTagDef[] = this.scriptTags.getHeadTags(scriptTagData);
+        const allTags: HeadTagDef[] = metaTags.concat(scriptTag);
+
+        // Concatenate meta and script tags if both, only meta tags or none
+        if (metaTags && scriptTag) {
+            this.set('headTags', allTags);
+        } else if (metaTags) {
+            this.set('headTags', metaTags);
+        }
```

Comment:
Because they are not meta tags and this format makes the code difficult to read and update. An all tags array is much more representative of the actual data being built, and should allow for others to better understand what is occuring in the code.

Reply:
So... rename to `allTags` and push the element? I am not sure how this is more readable. And suppose we have another type of tags that we need to add. Does that mean we need `metaTags`, `scriptTags`, `linkTags` and etc.? Conditionally concatenating these array into `allTags` would quickly become unfeasible.

Also, because empty arrays are truthy.